fiftyone/server/routes/sample.py

```
@@ -340,18 +351,19 @@ def handle_json_patch(target: Any, patch_list: List[dict]) -> Any:
             detail=f"Failed to parse patches due to: {err}",
         ) from err
 
-    errors = {}
+    errors = []
     for i, p in enumerate(patches):
         try:
             p.apply(target)
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)